src/test/kotlin/io/github/mojira/arisa/modules/CrashModuleTest.kt

```diff
@@ -29,6 +29,19 @@ const val EXAMPLE_CRASH = "---- Minecraft Crash Report ----\n" +
         "\tJava Version: 1.8.0_131, Oracle Corporation\n" +
         "Is Modded: Probably not. Jar signature remains and client brand is untouched.\n"
 
+const val SERVER_UNMODDED_CRASH = "---- Minecraft Crash Report ----\n" +
+        "// Daisy, daisy...\n" +
+        "\n" +
+        "Time: 6/28/17 11:44 AM\n" +
+        "Description: Initializing game\n" +
+        "\n" +
+        "org.lwjgl.LWJGLException: Pixel format not accelerated\n" +
+        "-- System Details --\n" +
+        "Details:\n" +
+        "\tMinecraft Version: 1.8.9\n" +
+        "\tJava Version: 1.8.0_131, Oracle Corporation\n" +
+        "Is Modded: Unknown\n"
```

Comment:
It'd probably be better to use a message from an actual server (you're not going to get an LWJGL crash on a server, for instance, and the actual "Is Modded" value isn't the exact string `Unknown`).  [MC-131071](https://bugs.mojang.com/browse/MC-131071) has some server crashes, and I'm sure there are other ones that can be used (possibly from the watchdog)

Reply:
I tested https://bugs.mojang.com/secure/attachment/178719/crash-2018-08-02_12.46.39-server.txt but they are quite big to hardcode into the test, got any suggestion?

Reply:
I think just this would be good (possibly even dropping the JVM flags, memory, profiler, and player parts), from [this report](https://bugs.mojang.com/secure/attachment/169864/crash-2018-06-05_21.20.43-server.txt):

```
---- Minecraft Crash Report ----
// Oh - I know what I did wrong!

Time: 6/5/18 9:20 PM
Description: Exception generating new chunk

java.util.concurrent.ExecutionException: java.lang.RuntimeException: We are asking a region for a chunk out of bound | -174 8

-- System Details --
Details:
	Minecraft Version: 1.13-pre1
	Operating System: Linux (amd64) version 4.4.0-98-generic
	Java Version: 1.8.0_151, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 614132352 bytes (585 MB) / 988282880 bytes (942 MB) up to 3340763136 bytes (3186 MB)
	JVM Flags: 2 total; -Xmx3584M -XX:MaxPermSize=256M
	Profiler Position: N/A (disabled)
	Player Count: 1 / 20; [so['KawaiiMutton'/351, l='Vanilla', x=8.50, y=72.00, z=121.50]]
	Data Packs: vanilla
	Is Modded: Unknown (can't tell)
	Type: Dedicated Server (map_server.txt)
```

Reply:
all done